.gitignore

```diff
@@ -5,6 +5,7 @@
 /bin/
 /Build/
 /build/
+/build-*/
```

Comment:
Might want to add the appimage deploy scripts here as well (or create a `.gitignore` in the appimage directory for them)